### PR TITLE
Add Complete Set component

### PR DESCRIPTION
**Pull request by iryb**
Task https://app.asana.com/0/1202525679663009/1202838302178341

complete-set-table

### CODE REVIEW

**Comment by iryb** [comment]
What do you think about white color box? I styled it as in mockup but the border is almost invisible on un-hover table row. Do you want to leave it as it is now or apply darker border?

**Comment by oleksandr-danylchenko** [comment]
> What do you think about white color box?

It should be framed into the `box-shadow`, not the `border`. The result can look smth like this - https://playcode.io/946847/

File: frontend/src/common/enums/car/options.enum.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  Complectation: 'Complectation',
+  Color: 'Color',
+  Motor: 'Motor',
+  WheelDrive: 'Wheel Drive',
+  Race: 'Race',
+  Price: 'Price',
+  Compare: 'Compare',
+};
+
+export { CarSetOptions };
```
Comment (oleksandr-danylchenko) [inline frontend/src/common/enums/car/options.enum.ts]: This is not `enum`. Why it's located in the `enum` file? :eyes:

File: frontend/src/components/complete-set-table/complete-set-table.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                {car.brand} {car.model}
+              </TableCell>
+              <TableCell className={styles.tableRow}>
+                {car.complectation}
+              </TableCell>
+              <TableCell className={styles.tableRow}>
+                <div
+                  className={styles.colorBox}
+                  style={{ backgroundColor: car.color }}
+                ></div>
```
Comment (oleksandr-danylchenko) [inline frontend/src/components/complete-set-table/complete-set-table.tsx]: `div` can be collapsed: 
```suggestion
                <div
                  className={styles.colorBox}
                  style={{ backgroundColor: car.color }}
                />
```

**Review by oleksandr-danylchenko — CHANGES_REQUESTED** [review]
- The colors boxes should be framed into the `box-shadow`
- We mustn't use objects instead of the `enum`s in TS

File: frontend/src/common/enums/car/options.enum.ts (hunk 1 above)
Comment (iryb) [reply frontend/src/common/enums/car/options.enum.ts]: You are totally right. Thank you!

**Review by oleksandr-danylchenko — APPROVED** [review]

**Pull request merged**